ui/ui-kit/Modal/Modal.tsx

```diff
@@ -0,0 +1,30 @@
+import s from "./modal.module.scss"
+
+type ModalType = {
+    active: boolean
+    setActive: any
+    title: string
+    children: any
+  }
+
+const Modal: React.FC<ModalType> = ({active, setActive, title, children}) => {
+    const classs = `${s.modal} ${s.effect}`
```

Comment:
try using the class name library, without it it's harder to understand the style logic